app/saml2/index.php

```diff
@@ -0,0 +1,81 @@
+<?php
+# verify php build
+include('functions/checks/check_php_build.php');		# check for support for PHP modules and database connection
+define("TOOLKIT_PATH", '/var/www/html/phpipam/functions/php-saml/');
```

Comment:
Hardcoded path?
Maybe fix this with:
`define("TOOLKIT_PATH", dirname(__FILE__).'../../functions/php-saml/');`